pkg/api/vlabs/validate_test.go

```diff
@@ -704,6 +704,10 @@ func Test_Properties_ValidateNetworkPluginPlusPolicy(t *testing.T) {
 			networkPlugin: "azure",
 			networkPolicy: "flannel",
 		},
+		{
+			networkPlugin: "azure",
```

Comment:
this combination is already in `networkPluginPlusPolicyAllowed ` in [validate.go](https://github.com/Azure/aks-engine/blob/master/pkg/api/vlabs/validate.go#L71) while this is adding it to the unit test expecting it to not be valid. Unless I'm missing something, it seems contradictory.